exporter/honeycombmarkerexporter/config.go

```diff
@@ -24,33 +27,35 @@ type Config struct {
 
 	// Markers is the list of markers to create
 	Markers []Marker `mapstructure:"markers"`
+
+	// DatasetSlug is the endpoint that specifies the Honeycomb environment
+	DatasetSlug string `mapstructure:"datasetslug"`
```

Comment:
The dataset slug should be specific to a marker.  Since a user can create multiple types of markers with a single exporter, we should allow them to specify where each specific marker lives.

For default value, we could either require the value be set if a marker is configured or set the value to `__all__` inside the Validate function.